services_app/src/main/res/layout/password_dialog_layout.xml

```diff
@@ -72,7 +80,9 @@
             android:layout_marginTop="5dp"
             android:layout_weight="1"
             android:text="@string/ok"
-            android:textSize="14sp"/>
+            android:textSize="14sp"
+            style="@style/ButtonStyle"
+            />
     </LinearLayout>
 
 </RelativeLayout>
```

Comment:
This is a very small issue. GitHub does not like a file that does not end in a blank line. So if it doesn't end in a blank line a red circle with a dash in it will appear point it out.

Reply:
Oh had no idea github have problems with blank lines. I'll clean that up in the next PR